tests/wpt/web-platform-tests/html/semantics/embedded-content/media-elements/event_timeupdate_noautoplay.html

```diff
@@ -17,7 +17,10 @@
 test(function() {
   var t = async_test("calling play() on a sufficiently long audio should trigger timeupdate event", {timeout:5000});
   var a = document.getElementById("a");
-  a.addEventListener("error", t.unreached_func());
+  a.addEventListener("error", function(e) {
+    console.log("===> ERROR", e.target.error.message, e.target.error.code);
+    t.unreached_func();
```

Comment:
This is not equivalent - unreached_func returns a function. You'll want `"error", t.step_func(function(e) { ... t.assert_unreached() })`